### ISSUE TITLE
Typecasting @var should not be multiline.

### ISSUE THREAD

**Issue by haringsrob** [squizlabs/PHP_CodeSniffer#2097]
According to what I can find on the web, the standards imply that we should use:

```php
function example()
{
    /** @var Example $test */
    $test;
}
```

However currently when I am running with PSR2 this is being reformatted to:

```php
function example() 
{
    /**
 * @var Example $test 
*/
    $test;
}
```

Related issues that I could find are:
https://github.com/squizlabs/PHP_CodeSniffer/issues/1479
https://github.com/squizlabs/PHP_CodeSniffer/issues/359

But I think those are about class variables.

fig link:
https://github.com/php-fig/fig-standards/blob/0e075501278402429b30b4a86b0fa736c59b5f35/proposed/phpdoc.md#3-definitions

```
"DocComment" is a special type of comment which MUST

start with the character sequence /** followed by a whitespace character
end with */ and
have zero or more lines in between.
```

**Comment by gsherwood** [comment]
When I run PHPCS over your sample code using the PSR2 standard, I don't get any errors or any fixes. The PSR2 standard specifically excludes comments (of all types) so PHPCS shouldn't be reporting errors for comment formatting.

Are you sure you are using the PSR2 standard to produce these errors?

> fig link:
> https://github.com/php-fig/fig-standards/blob/0e075501278402429b30b4a86b0fa736c59b5f35/proposed/phpdoc.md#3-definitions

Note that PSR-5 has been abandoned. I was hoping it would be completed so there was a consistent doc standard I could get PHPCS to enforce, but that wasn't to be. Hopefully someone will pick it up and keep moving forward with it, but I won't be trying to enforce any rules until it is at least in the review stage.

**Comment by haringsrob** [comment]
Thanks for the reply. I think this was due to misconfiguration.

I was unable to reproduce using cli only.

Closing this.
